package.json

```diff
@@ -45,7 +45,8 @@
     "commitlint": "commitlint --from origin/main",
     "format": "prettier --write .",
     "clean": "node ./scripts/clean.js",
-    "docs": "lerna run --stream maybe-install-deps-and-build-docs"
+    "docs": "lerna run --stream maybe-install-deps-and-build-docs",
+    "fixbuild": "pushd packages/worker && rm -r ./lib/*.d.ts || tsc --build && popd && lerna run --stream build"
```

Comment:
:(

Reply:
I use `npm run clean -- --only ts && npm run build`

Reply:
Well that would've been nice to know! :stuck_out_tongue_closed_eyes: